lib/enhancements/edition.rb

```diff
@@ -140,4 +140,13 @@ def artefact
   def self.convertible_formats
     Artefact::FORMATS_BY_DEFAULT_OWNING_APP["publisher"] - ["local_transaction"]
   end
+
+  def fact_check_id
```

Comment:
I don't understand why this needs to be deterministic, rather than just using SecureRandom.